doc/source/index.rst

```diff
@@ -235,6 +235,7 @@ Getting Involved
    :maxdepth: -1
    :caption: Ray Core
 
+   walkthrough.rst
```

Comment:
Moved walkthrough to top level.

Reply:
<img width="1118" alt="Screen Shot 2020-02-09 at 2 53 48 PM" src="https://user-images.githubusercontent.com/4529381/74111767-fb5c9300-4b4b-11ea-85ec-88e41f1fd162.png">